libs/component-sample/src/lib/feature/collapse-sample/collapse-sample.component.html

```diff
@@ -1,5 +1,17 @@
-<sds-collapse
-[buttonLable]="buttonText"
-[collapseContent]="collapseContentInput"
-></sds-collapse
-  >
+ <div class="text-center">
+  <button type="button"
+  class="usa-button text-center"
+  [attr.aria-expanded]="!isCollapsedContent"
+  aria-controls="collapseID"
+  (click)="isCollapsedContent = !isCollapsedContent"
+  > Show / Hide Content
+  </button>
+ </div>
+
+
+<p
+  id="collapseID"
+  [sdsCollapse]="isCollapsedContent"
+   class="usa-accordion__content bg-base-lighter margin-top-1">
```

Comment:
Let's not use usa-accordion__content class in the example. It's not incorrect but it may confuse the implementing developer that this is being used for collapse.